index.html

```diff
@@ -0,0 +1,94 @@
+      <li class="list-group-item rounded-lg">Zuckerberg was not a fan of The Social Network, arguing that many of the film's details were inaccurate.</li>
+      <li class="list-group-item rounded-lg">Zuckerberg notes that the film took huge liberties with some events, while staying oddly true to others. He observed that while he never had any interest in joining the finals clubs (an element that plays a major role in the film), every shirt and fleece depicted in the movie was an actual piece of clothing that he has owned.</li>
+      <li class="list-group-item rounded-lg">The Social Network is based off of the book, The Accidental Billionaires, by Ben Mezrich. The book has been known to deviate quite a bit from actual events, as even the book's publicist referred to it as "big juicy fun" rather than "reportage."</li>
+      <li class="list-group-item rounded-lg">The Social Network's screenwriter, Aaron Sorkin, confessed to New York magazine that, "I don't want my fidelity to be the truth; I want it to be storytelling...what is the big deal about accuracy purely for accuracy's sake?"</li>
+      <li class="list-group-item rounded-lg">Mark Zuckerberg was one of the early adopters of Google+, saying that it validated his vision of mainstream social networking. In 2011, Zuckerberg was the most followed user on Google+, with even more followers than Google co-founders Larry Page and Sergey Brin.</li>
+      <li class="list-group-item rounded-lg">Zuckerberg has supported numerous philanthropic causes, most famously donating $100 million to rescue the struggling Newark Public Schools system of New Jersey.</li>
+      <li class="list-group-item rounded-lg">Zuckerberg has signed the "Giving Pledge" commitment, promising to donate at least 50% of his wealth over the course of his lifetime.</li>
+      <li class="list-group-item rounded-lg">In 2014, Zuckerberg donated $25 million to combat the ebola virus epidemic in West Africa.</li>
+      <li class="list-group-item rounded-lg">Zuckerberg's marriage to longtime girlfriend Priscilla Chan was quite the surprise for friends and family--guests were told they were being invited over to celebrate Chan's medical school graduation, but in reality they ended up attending the pair's wedding.</li>
+      <li class="list-group-item rounded-lg">Similar to Steve Jobs, Mark Zuckerberg has a $1 annual salary as CEO of Facebook.</li>
+      <li class="list-group-item rounded-lg">Zuckerberg was Time magazine's "Person of the Year" in 2010, Facebook made the Fortune 500 list for the first time--making Zuckerberg, at the age of 28, the youngest CEO on the list.</li>
+      <li class="list-group-item rounded-lg">Zuckerberg is currently #5 on Forbe's list of the 400 richest Americans.</li>
+    </ul>
+
+
+
```

Comment:
Avoid unnecessary spacing in your code to improve the readability

Reply:
- [ ] Good job using inline comments here 👌 However, it's important to ask the student to make the change more politely. using words like please, kindly, I suggest, and I think it would be better ...
> Instead of giving orders make suggestions 💯

![image](https://user-images.githubusercontent.com/45231276/178229738-36cdb199-b2f9-470a-9a79-3c5af6c728c2.png)